src/main/resources/modules/uti/abx_tx.json

```diff
@@ -0,0 +1,1080 @@
+{
+  "name": "Abx_Tx",
+  "remarks": [
+    "A blank module"
+  ],
```

Comment:
Should probably just delete these rather than leave the default "A blank module"